README.md

```diff
@@ -22,121 +22,112 @@ refreshing the code and making it available for easy reuse by others._
 
 ## Architecture
 The model is written in Python, using PyTorch for the CNN. We provide 3 separate
-"stages", which are run using different commands and arguments:
+"steps", which are run using different commands and arguments:
 
 * data processing: downloads part of CM2.6 dataset and processes
 * model training: train model on processed data
 * model testing: tests the trained model on an unseen region
 
-For more details on each of the stages, see the [`docs`](docs/) directory.
+For more details on each of the steps, see the [`docs`](docs/) directory.
 
 ## Usage
 ### Dependencies
-Python 3 is required.
+Python 3.9 or newer is required.
 
 #### Python
 With `pip` installed, run the following in the root directory:
 
     pip install -e .
```

Comment:
I know it's not a current change, but maybe it would be clearer to first explain the virtual environment step and then the pip install, so that the commands are all listed in the correct order here

Reply:
Made that change in main.